mac_client.md

```diff
@@ -0,0 +1,93 @@
+
+To use the Mac client, you must perform some setup on both the your Mac and Podman nodes. In this case,
+the Mac node refers to the Mac on which Podman is being run; and the Podman node refers to where
+Podman and its storage reside.
+
+### Connection settings
+Your Linux box must have ssh enabled, and you must copt your Mac's public key from `~/.ssh/id.pub` to
+`/root/.ssh/authorized_keys` on your Linux box. This allows for the use of SSH keys for remote access.
+
+You may need to edit your `/etc/ssh/sshd_config` in your Linux machine as follows:
+```
+PermitRootLogin yes
+```
+
+Use of SSH keys are strongly encouraged over regular login. However, if you wish toavoid ‘logging in’ every
+time you run a podman command, you may edit your `/etc/ssh/sshd_config` on your Linux machine as follows:
+```
+PasswordAuthentication no
+PermitRootLogin without-password
+```
+
+### Podman node setup
+The Podman node must have Podman (not the Mac client) installed as normal. If your system uses systemd,
+then simply start the Podman varlink socket.
+```
+$ sudo systemctl start io.podman.socket
+$ sudo systemctl enable io.podman.socket
+```
+
+If your system cannot use systemd, then you can manually establish the varlink socket with the Podman
+command:
+```
+$ sudo podman --log-level debug varlink --timeout 0  unix://run/podman/io.podman
+```
+
+### Required permissions
+For now, the Mac client requires that you be able to run a privileged Podman and have privileged ssh
+access to the remote system.  This limitation is being worked on.
+
+
+#### Running the remote client
+There are three different ways to pass connection information into the client: flags, config file, and
+environment variables. All three require information on username and a remote host ip address. Most often,
+your username should be root and you can obtain your linux-ip using `ip addr`
+
+To connect using flags, you can use
+```
+$ podman --remote-host linux-ip --username root images
+REPOSITORY                     TAG      IMAGE ID       CREATED         SIZE
+docker.io/library/ubuntu       latest   47b19964fb50   2 weeks ago     90.7 MB
+docker.io/library/alpine       latest   caf27325b298   3 weeks ago     5.8 MB
+quay.io/cevich/gcloud_centos   latest   641dad61989a   5 weeks ago     489 MB
+k8s.gcr.io/pause               3.1      da86e6ba6ca1   14 months ago   747 kB
+```
+If the conf file is set up, you may simply use podman as you wold on the linux machine. Take a look at
+[podman-remote.conf.5.md](https://github.com/containers/libpod/blob/master/docs/podman-remote.conf.5.md) on how to use the conf file:
+
+```
+$ podman images
+REPOSITORY                     TAG      IMAGE ID       CREATED         SIZE
+docker.io/library/ubuntu       latest   47b19964fb50   2 weeks ago     90.7 MB
+docker.io/library/alpine       latest   caf27325b298   3 weeks ago     5.8 MB
+quay.io/cevich/gcloud_centos   latest   641dad61989a   5 weeks ago     489 MB
```

Comment:
remove this name.  Perhaps add some redhat images.

Reply:
quay.io/podman/stable  perhaps?